test/parallel_api/iterator/transform_iterator.pass.cpp

```diff
@@ -60,8 +60,6 @@ struct test_copy_if
 
     template <typename ExecutionPolicy, typename Iterator1, typename Iterator2>
     void operator()(ExecutionPolicy&& /* exec */, Iterator1 /* first1 */, Iterator1 /* last1 */, Iterator2 /* first2 */) {
-        using value_type = typename ::std::iterator_traits<Iterator1>::value_type;
```

Comment:
BTW, Is whether the `struct test_copy_if` used somewhere?
I could find the usage in this test file...

Reply:
test2buffers<alloc_type, test_copy_if<ValueType>>();

Reply:
Removed